app/views/collections/edit.html.erb

```diff
@@ -14,6 +14,9 @@
         <%= f.text_area :description, :class=>"form-control rich-text-edit", :rows => 5 -%>
       </div>
 
+      <%= render :partial => 'extended_metadata/extended_metadata_type_selection', :locals => {:f => f,:resource => @collection } %>
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys

Reply:
yes,  i know `:partial=>` is deprcated, the only reason i was using it to make consistent style with the rest of code.